markup.go

```diff
@@ -160,6 +161,34 @@ func Class(class ...string) Applyer {
 	})
 }
 
+// mustValidateClassNames ensures no class names have spaces
+// and panics with clear instructions on how to fix this user error.
+func mustValidateClassNames(class []string) {
+	for _, name := range class {
+		if containsSpace(name) {
+			failClassName(name)
```

Comment:
Remove `failClassName` and just do:

```Go
panic(`vecty: invalid argument to vecty.Class "` + name + `" (string may not contain spaces)`)
```

?

Reply:
@slimsag makes sense.